### ISSUE TITLE
Support for autocomplete based on schema

### ISSUE THREAD

**Issue by onx2** [11bit/zed-extension-graphql#7]
I'm using an extension in VSCode that gives me autocomplete in `.graphql` files based on a configuration file: `.graphqlrc.ts`. I was wondering if this is a feature you would be willing to add.

Thanks for this extension!

**.graphqlrc.ts**
```ts
export default {
  schemaPath: "src/schema.generated.json",
  documents: "**/*.graphql",
};
```

**Example of autocomplete**
<img width="321" alt="image" src="https://github.com/user-attachments/assets/01b8e374-816f-4df1-883d-2c1452277765">

**Comment by affanshahid** [comment]
Would really appreciate if this gets added

**Comment by 11bit** [comment]
Yes, I have it my mind. Unfortunately, I don't have time right now. Hopefully I can come back to it in late Aug or Sep

**Comment by onx2** [comment]
Got it - do you want me to leave this open for tracking or close? @11bit

**Comment by 11bit** [comment]
Let's keep it open. Maybe somebody else would like to contribute

**Comment by 11bit** [comment]
Implemented in #8
